source/manual/howto-modify-change-note.html.md

```diff
@@ -28,3 +29,51 @@ $ d.live.change_note.update(note: "Foo")
 $ d
 => #<ChangeNote id: 397621, note: "Foo", public_timestamp: "2020-06-16 11:02:22", edition_id: 5146461, created_at: "2020-06-16 10:22:34", updated_at: "2020-06-16 15:24:53">
 ```
+
+## Whitehall
+
+Connect to the Whitehall console (test your change on Integration first):
+
+```
+govuk-connect app-console -e integration whitehall_backend/whitehall
+```
+
+This finds the document, finds the correct edition within it, and updates it. You need the document ID, a string to search for to identify the right change log entry (search is fuzzy, be precise to get the right one), and the updated content
```

Comment:
Whats the first "This" referring too? (`This finds the document...`). I was slightly confused because I thought you meant the command above.

Reply:
Ah, it's the whole list of commands, I'll update to clarify.